sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -192,17 +196,34 @@ class AccountStateView(private val metadataStorageView: AccountStateMetadataStor
     val processor = messageProcessors.find(_.canProcess(message, this)).getOrElse(
       throw new IllegalArgumentException(s"Transaction ${ethTx.id} has no known processor.")
     )
-    val gasUsed: BigInteger = processor.process(message, this) match {
+
+    val consensusDataReceipt : EthereumConsensusDataReceipt = processor.process(message, this) match {
       case success: ExecutionSucceeded =>
-        success.gasUsed()
+        val evmLogs = getLogs(txHash)
+        val logs = evmLogs.map(element => new EthereumLog(
+          new EthereumConsensusDataLog(element)))
+        val gasUsed = success.gasUsed()
+        new EthereumConsensusDataReceipt(
+          ethTx.version(), ReceiptStatus.SUCCESSFUL.id, prevCumGasUsed.add(gasUsed), logs)
```

Comment:
Good example here.
We may use the EvmLog class directly, no need to wrap it by `EthereumConsensusDataLog`.

Reply:
Ok